deployment/E_heroku.md

```diff
@@ -1 +1,308 @@
-## FPO
+### What we'll need
+
+The only thing we'll need for this guide is a working Phoenix application.
+
+If you don't already have one, just run `mix phoenix.new hello_phoenix` in a terminal and you're good to go.
+
```

Comment:
I think it would be a good idea to prompt readers to `git init` their project right here if they haven't yet done so. It might also be useful to say that the reason we're doing that is the style of Heroku deployment we'll be doing is based on git.